media/js/base/mozilla-client.js

```diff
@@ -27,8 +100,20 @@ if (typeof Mozilla === 'undefined') {
     };
 
     /**
+     * Get the user's Firefox major version number. 0 will be returned on non-Firefox browsers.
+     *
+     * @private
+     * @param  {String} ua - browser's user agent string, navigator.userAgent is used if not specified
+     * @return {Number} major version number
+     */
+    Client._getFirefoxMajorVersion = function (ua) {
+        return parseFloat(Client._getFirefoxVersion(ua));
```

Comment:
I was actually thinking old [releases](https://www.mozilla.org/en-US/firefox/releases/) like 3.6 but yes, 38.4.0 should be considered as `38`, not `38.4`... I'll use `parseInt` instead.